_shared/project/conf/supervisord-dev.conf

```diff
@@ -0,0 +1,43 @@
+[supervisord]
+nodaemon = true
+silent = true
+{% if cookiecutter.get("_directory") == "pyapp" %}
+
+[program:app]
+command=python3 -m {{ cookiecutter.package_name }}
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+{% endif %}
+{% if cookiecutter.get("_directory") == "pyramid-app" %}
+
+[program:web]
+command=gunicorn --bind :{{ cookiecutter.port }} --workers 1 --reload --timeout 0 --paste conf/development.ini
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+{% endif %}
```

Comment:
This is the `web` program that the `pyramid-app` cookiecutter's `supervisord-dev.conf` already contains on `main`.